### PR TITLE
Holoparasite nerf

### PR DESCRIPTION
**Pull request by LanCartwright**
Made using recent statistics and heavy player feedback.
Raises the cost by 4, from 12 to 16 tc.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
i've been telling kor to do this for awhile, they fucking need it

**Comment by Anonus** [comment]
Very, very debatable. :-1:

**Comment by KorPhaeron** [comment]
@LanCartwright 

Please refer to https://github.com/tgstation/-tg-station/pull/13710

**Comment by Bawhoppen** [comment]
It still needs it even if it can't appear in lowpop

**Comment by CPTANT** [comment]
I say 14

**Comment by LanCartwright** [comment]
@CPTANT Will consider it.

**Comment by Bawhoppen** [comment]
15 so they can't get an emag but still something else

**Comment by KorPhaeron** [comment]
>15 so they can't buy the almost required traitor item

??

**Comment by LanCartwright** [comment]
yeah if you get 2v1 undroppable esword parasite you don't get the emag.

**Comment by Bawhoppen** [comment]
so they can't get an already ridiculously OP item AND a guardian that can infi-setfire and teleport people

**Comment by KorPhaeron** [comment]
>MUH 2v1

Better remove nuke, wizard apprentices, rev, gang, cult, HoG, xenomorphs, cyborgs, xenobio, constructs, syndi radio headsets, changelings, shadowling, codewords and necrostone while you're at it

>esword

Esword does 50% more damage, blocks attacks, has a 30% chance of knocking you down, and bypasses shields.

If you put on riot armour+hold a shield the strongest parasite does an average of 6 damage per hit against you with 0 knockdown chance.

**Comment by LanCartwright** [comment]
kor its a unstunnable adamantite golem inside of you. it's stealthy as fuck. there is no balancing that comparitively.

**Comment by KazeEspada** [comment]
Kor, fire parasite beats riot armor. Since fire bypasses armor. Atmos Hardsuit is the best defense.

**Comment by Bawhoppen** [comment]
What if we did stands at 14 but increased emag to 8? It would balance alot of other things too.

**Comment by KorPhaeron** [comment]
Except it can't stun you and shares your health pool/doubles your "hitbox." Its absolutely destroyed by laser weapons

>golem

It shares almost nothing in common with golems

>kor it does esword damage!!
>actually it doesnt
>WELL WHAT ABOUT THIS OTHER TYPE?

Every conversation about parasite balance turns into "well they can do all these things" as if every ability is shared by one type.

**Comment by Bawhoppen** [comment]
You have to admit it is pretty damn imbalanced Kor just from seeing it in practice.

**Comment by KazeEspada** [comment]
The problem with balancing the parasite is that we are not balancing 1 parasite but at least 5(cant recall all forms off the top of my head).

**Comment by tkdrg** [comment]
Just split the emag into two items. The biggest issue here is emags, and not stands. I could see parasites at maybe 13 but not fucking 16.

**Comment by AlexanderUlanH** [comment]
Traitors are deadly after a concerted effort to make them deadlier.  Fix posthaste.

**Comment by Robustin** [comment]
Man the fuck up.

The HP is a crucial crutch for bad traitors (Hint: A majority of traitors are NOT ROBUST). It allows even the most idiotic of traitors to be carried by someone who, hopefully, knows what they are doing. It solves a problem that's existed since we nerfed sleepypens, namely how do we make the "average" traitor a force to be reckoned with a not a loot pinata for security?

The holoparasite is an elegant solution that can make shitty traitors deadly without relying on uncounterable mechanics.

I cannot fathom what is going through anyone's heads in support of any kind of nerf for this. The cost still prohibits any cheesy ebow + HP combo, but still lets you get stuff that can be critical like emags.

BAD

**Comment by KorPhaeron** [comment]
>The holoparasite is an elegant solution

I do like how pulling a ghost usually means you have an oldfag mentor who can explain game mechanics/common strategies/give the user a bit of confidence to go do stuff

**Comment by Durkel** [comment]
Good god no, Stop trying to nerf holo's already. What's next, a revolver nerf?

**Comment by Bawhoppen** [comment]
just because you like an unbalanced tdm gamemode doesn't mean it should be that way

**Comment by Iamgoofball** [comment]
Do what I did:

Zap the parasite owner with the tesla engine

**Comment by Zilenan91** [comment]
To be honest, parasites are an objectively better esword. They always block taser shots like eswords sometimes do, can't be stunned like esword users can be, can break walls, and can have healing + teleporting for only 5 less damage, all on the same item as well as huge utility and gimmick options in the form of alternate parasites that aren't standard/support.

**Comment by Shapsy** [comment]
>The HP is a crucial crutch for bad traitors (Hint: A majority of traitors are NOT ROBUST)

subtle bragging


Holoparasites are fine as they are now. They draw literally the entire station's attention towards you and get absolutely dunked on by any sort of ranged attack, or just stunning the traitor and dragging them away from their stand while clubbing them in the head. Not only that, it literally disintegrates you on death, so unless you scanned yourself in genetics beforehand (even then you'd lose all your gear), once someone manages to kill you, that's it for you. 

Fighting holoparasites the correct way (AKA with a team of people with laser guns, and not running after them with a spear alone in maint) is a lot of fun since it requires coordination and usually some sort of shootout, I'd rather fight someone with a holoparasite than someone running around with a revolver.

>To be honest, parasites are an objectively better esword. They always block taser shots like eswords sometimes do, can't be stunned like esword users can be,

Then stun the non stun-resistant human and drag them away while beating them, the holoparasite can't catch up

**Comment by Zilenan91** [comment]
But if you stun a dude with an esword they're fucked anyways, that's no different for a holo.

**Comment by Shapsy** [comment]
Then if it's still perfectly possible to kill them, then what's the issue?

**Comment by Zilenan91** [comment]
Because they're objectively better than even a double esword in every way, and have far more utility. They can eat tasers, break walls, make you take 50% less damage, drag you away from death at the speed of sound, and only do 1/3 less damage than the dual esword. The only downside is they can't block lasers but if there's just a singular dude with a laser against you you can just prod him and rek him with your parasite. Any more and you die, same as dual eswords, really.

**Comment by Shapsy** [comment]
They can't block tasers if you shoot the guy instead though? I don't know why you think it's impossible to tase the stand user without hitting the stand, the stand can't be on all sides at once

And one laser gun is more than enough to dump into a parasite while backpedaling to put their owner into medium/deep crit, which is basically death for the parasite user unless they have access to medical supplies and cryo for their cellular damage

**Comment by Zilenan91** [comment]
Shoot a stand user down a hallway. I can guarantee you the stand will pop out and start body blocking you hardcore.

Also you need to spam literally the entire ammo of a laser gun JUST to put someone into crit if the standard holo gets hit by all of them. No genetic damage, maybe some bleeding, and that's it.

**Comment by KorPhaeron** [comment]
>standard holo gets hit by all of them. 

So shoot the user instead. If the tanky melee only creature is soaking all your hits then its not going to be as effective

>To be honest, parasites are an objectively better esword. 

They also cost more

**Comment by LanCartwright** [comment]
They're also undisarmable, completely stealthy, and allows you to do something else whilst it's doing it's job.

Also you can fight along it.

**Comment by The-Albinobigfoot** [comment]
:-1:
Nice to see the Balance Brigade is alive and well.

**Comment by KorPhaeron** [comment]
If you truely believe they are monstrously unfair to fight I am not sure what making them cost more will do anyway. You'll just he killing any creativity/flexibility on the part of the traitor in gear selection/planning they'll be just as strong and just as "impossible" to deal with after this

**Comment by Shapsy** [comment]
How are they stealthy if having one pretty much makes you KOS for everyone on the station? Like, you can only surprise people with a holoparasite once or maybe twice before people hear you have a parasite

**Comment by newfren** [comment]
guys you can hide the esword just by turning it off the holoparasite isn't alone in being stealthy

see: every traitor item except like... the hardsuit

**Comment by Shapsy** [comment]
If anything the holoparasite is way less stealthy than the esword, cause good luck hiding the big red "Blood sprays from [summoner] as [src] takes damage!" message that flat out confirms that you're the one with the holoparasite to anyone watching

**Comment by LanCartwright** [comment]
Yeah well seccy sec officer can't search you and find an Esword. In fact he'll find a big "fuck you" in the form of a holoparasite that can kill or maim him, and afaik can uncuff you.

**Comment by KorPhaeron** [comment]
>and afaik can uncuff you.

Nope

>In fact he'll find a big "fuck you" in the form of a holoparasite

Walk away? Preferably dragging your cuffed prisoner and batoning them while laughing at the parasite

**Comment by LanCartwright** [comment]
Ah no more uncuffing good.

But think of the idea of someone fighting when the owner is distracted, unable to fight or similar.

But there's other matter to this balance. I've seen several rounds where someone has picked up brute and burn patches, and just spammed them on themselves whilst stunning as many as possible leaving the Holoparasite to do a majority of the work.

There is no way you can justify that an Esword, or two Esword are worth more than this.

The holoparasite literally does it all. This is the severity of the ramifications.


If you're so vehemently against changing the price I hope you understand these ramifications then let us move onto other ways to nerf it.

How about making stuns affect both of them; if one is stunned so is the other.

**Comment by PKPenguin321** [comment]
holoparasites aren't even that good except for 1v1s, and even then a traitor can still lose the 1v1 even with an active holoparasite
god damn you people need to get good :-1:

**Comment by KorPhaeron** [comment]
>Ah no more uncuffing good.

They never had it

**Comment by Dorsisdwarf** [comment]
Eswords are what, 6TC? 8TC?

They do more damage, can block shots, go straight through any station armor (Which is critical because it means actually they do WAY more than 30% extra damage against the people it really matters), and are 60% of the price of a parasite.

Their downside is that they only block shots 30% of the time, they can be disarmed, they can't drag you to medbay, and they can be searched for.

**Comment by KorPhaeron** [comment]
8tc, 50% blockrate, 50% more damage, ignore shields entirely, and have a 40% chance to knockdown on hit.

Pretty close though

**Comment by Jazaen** [comment]
Just make teleporting everyone from the escape shuttle by holo impossible, since that is probably worst part about that.

**Comment by Aloraydrel** [comment]
you ded tbh

**Comment by LanCartwright** [comment]
I don't think anyone understands. This is literally an item that when bought is better than any other.

It's hidden inside you and unless someone has lethals, they cannot get to you. 

Here's a problem, you can stun someone and begin to kill them. Not only is the AI on your side if someone tries to kill you, but so is the holoparasite.

If the crew wishes to stop you, they would be using lethals because as we've discussed and the AI under asimov is required to protect the human host.

THE HOLOPARASITE IS STUN IMMUNE.
Non-lethal takedowns can literally be stopped by the holoparasite popping out and blocking a stun. In a 1 square wide hall anything except lethals DO NOT WORK because the holoparasite can and will block it. If your back is against the wall, the holoparasite will prevent stunning of any kind.

**Comment by Bawhoppen** [comment]
I'm suggesting the stand should be more of a glass cannon than a massive damage tank, make it more like a wraith

**Comment by LanCartwright** [comment]
Do you suggest giving the Holoparasite less health or amplify damage taken by the Holoparasite to the owner?

**Comment by Bawhoppen** [comment]
The stand doesn't have it's own health, but shares the host's health. So amplify it's damage. But there is 5 different types so we'd need indivual solutions for each of them

**Comment by MMMiracles** [comment]
A holoparasite is good for 1v1 and sometimes crowd control (chaos memes reign supreme). A prepared security force can absolutely DESTROY a holoparasite user.

Standard? Throw shit at it/shoot it till the owner drops. Laugh as the parasite tries to do a final stand to protect its master.

Chaos? Laugh as literally anyone with a firesuit beats the shit out of it in melee with little to no harm done to the attacker.

Support? Don't let it stand still to recall the user and a fire extinguisher out-damages the healing.

Ranged? CRYSTAL SPRAY is pathetic anyone why did you get this.

Explosive? Don't touch the thing the parasite last touched, beat it up as per usual.


Tl;dr:
g i t  g o o d
i
t

g
o
o
d
:-1:

**Comment by Bawhoppen** [comment]
@MMMiracles have you ever joined the server in a traitor round? its not about 'gitting gud'

**Comment by MMMiracles** [comment]
I've been the stand, the owner, and the person on the resulting end. 

It is about gitting gud.

**Comment by Bawhoppen** [comment]
Just because a thing is physically killable, doesn't mean its balanced.

**Comment by MMMiracles** [comment]
Did you miss the list I gave explaining the methods behind killing each stand or.

**Comment by Bawhoppen** [comment]
See my previous post

**Comment by MMMiracles** [comment]
Once again:

G I T
U
D

**Comment by Bawhoppen** [comment]
>ling is kill-able 

>must be balanced

**Comment by MMMiracles** [comment]
A holoparasite and its owner doesn't have infinite healing. 

Seriously people how bad can you be are you trying to 1v1 it with a fire extinguisher or.

**Comment by Bawhoppen** [comment]
I'm saying thats your logic.

**Comment by MMMiracles** [comment]
Anything can be taken down if people bother to group up against it. Ling isn't meant to be 1v1 able and neither is the stand. Ling might be silly at times but still. 

If you're 1v1ing at a time you deserve to die. Get Smart people like damn.

**Comment by LanCartwright** [comment]
Actually I've seen someone 5v1 with a holoparasite, stunprod and 2 healthkits.

I'm sorry that munchkinning is the only way to beat something. Don't you think it's a little ridiculous that we're expected to expect a stunprod or a stolen taser or these things

**Comment by MMMiracles** [comment]
Thank goodness we have a security force who has access to an entire armory of various ranged lethals who were probably told by the crew/ai that there is a stand.

**Comment by LanCartwright** [comment]
I think you need to drop the Ad-Hom bullshit and play sec against a Holoparasite. I don't think you understand balance at all.

For 14 TC you get:
An unstunnable mob.
Able to deal a fair amount of damage.
Stealthy (inside of you).
Able to drag you away.
Able to block shots and take shots.
Cannot drop it's weapons or be disabled.

In return:
You share the same health.
You turn to dust on death.


That's some incredibly overpowered shit. You cannot justify this at 14 TC as compared to the revolver, dual esword or any other similar item.

**Comment by MMMiracles** [comment]
I've been on every potential side and both won/lost. 

If I wanted pure murder potential I'd get the revolver+military belt. There's some gimmicks shit in parasites but you people don't seem to realize they AREN'T that powerful. 

Git gud.

**Comment by Bawhoppen** [comment]
@MMMiracles smh

**Comment by MMMiracles** [comment]
Fight me kiddo.

**Comment by LanCartwright** [comment]
Revolver can be disarmed and isn't immune to stun.
You can't have your holoparasite turned against you, you can with a revolver.
Revolvers can be found when searched, holoparasites can't.

**Comment by KorPhaeron** [comment]
Revolver is for instant death assasinations, it's a completely different niche. Unless you have an autolathe in which case it is doing a shit ton damage than stands can ever dream of doing.

>can disarm

Okay great. Does every fight in the game need the same strategy (stun/disarm)?

**Comment by MMMiracles** [comment]
If SS13 was 100% 'balanced' it'd be boring as fuck. 

We have rng stun punches that can make a naked man beat a guy in full riot armor. 

Parasites are fine as is and you people just need to learn to git gud.

**Comment by KorPhaeron** [comment]
Also yes oGame was a controlled game of numbers and competetive PvP with players on the same footing.

The nature of SS13 is not about straight fair match ups

**Comment by Bawhoppen** [comment]
It doesn't need to be perfectly balanced, but it needs to not be retardedly balanced.

**Comment by MMMiracles** [comment]
For over half of your tc you get a partner who hides in your body and shares your health. 

I'd sure hope it's capable of fucking people up 1v1 or else I would of just bought something else like a revolver or ebow to do the work myself. 

Literally every parasite besides support is fucked when their master hits crit for majority of situations. It doesn't matter if they have a healing virus due to clone loss stacking during crit. You'd need decent chems to be able to hop back up without being majority hurt by clone loss. 

Literally just do damage till the user falls down and let nature take over.

**Comment by Bawhoppen** [comment]
The problem is they are more effective in countless ways than their counters and weaknesses are, even something like a syndieborg has clear limitations

**Comment by MMMiracles** [comment]
Much like the syndieborg, it's effectiveness is tied to the skill of the player playing it. 

A competent borg can murder the whole crew singlehandedly or die to the first flash. 

A competent stand can be a viable partner in killing. It can also be a giant bullet sponge that soaks up the armory and gets you dusted within 5 minutes.

**Comment by Bawhoppen** [comment]
You just did my argument for me?

**Comment by KorPhaeron** [comment]
>stands are stronger than syndiborgs

Is this whole thing some elaborate shitpost

**Comment by MMMiracles** [comment]
This entire PR is a meme.

**Comment by Bawhoppen** [comment]
@KorPhaeron I didn't say they were necessarily stronger, but they're definitely much much stronger in some circumstances

**Comment by KorPhaeron** [comment]
>in some circumstances

????

The only situation in which stands are truly unfair right now is a chaos type on the escape shuttle

**Comment by Bawhoppen** [comment]
One example is borg is better for utility like opening doors, while stand is good for stealth maint snipes in maint and killing people in close quarters

**Comment by KorPhaeron** [comment]
How is it better at killing people in close quarters when the borg has grenades that kill you in one hit, an LMG with essentially infinite ammo that does more damage than the strongest parasite attack, and an esword

**Comment by Bawhoppen** [comment]
2 on one attacking, infinite stealth, less counters?

**Comment by MMMiracles** [comment]
Counters:
Dragging owner away while applying baton
Applying liberal use of laser
Syringe memes
Don't stand next to the stand

**Comment by Shapsy** [comment]
We've been over this, parasites aren't stealthy when they literally come with a big red message saying "HEY THIS GUY HAS A HOLOPARASITE" whenever the parasite gets hurt.

Literally the be-all end-all of this is that you guys need to get good at killing parasites. You can't tase the parasite? Laser it instead, or taser the stand user himself and drag him away while beating him. I've seen it done dozens of times with no issues. One dude laser gun is enough to put the parasite on the defensive, any more than that and the parasite stands no chance unless they and their user act really fast

**Comment by Bawhoppen** [comment]
@MMMiracles >implying its that simple

**Comment by MMMiracles** [comment]
It IS that simple. It's been done dozens of times. 

I shall return my original statement:
G I T
U 
D

**Comment by KorPhaeron** [comment]
I need to add stat tracking to prove parasites have a higher mortality rate than standard traitors don't I

**Comment by Bawhoppen** [comment]
you're the same kind of person who wants noskill sanic speed

**Comment by MMMiracles** [comment]
I don't want sonic speed what. 

I need more ways to say git gud,  help.

**Comment by Bawhoppen** [comment]
I don't want to list every single situation why its unbalanced

**Comment by tkdrg** [comment]
I think we've mostly agreed that holoparasites aren't OP, and that even if they were, this would not be the right way to nerf them. This discussion is just going around in circles at this point. So I'm going to go ahead and close this PR, feel free to talk with another maintainer or with our headcoder (cheridan) if you disagree.
